src/BaseBuilder.js

```diff
@@ -141,6 +142,20 @@ export default class BaseBuilder {
   }
 
   /**
+   * Set the order in the options object for this type.
+   *
+   * @param {transformer} order
+   * @return {Builder}
+   */
+  setOrder(order) {
+    invariant(
+      ['asc', 'desc'].indexOf(order) !== -1,
+      'Order must be either \'asc\' or \'desc\'',
```

Comment:
This is good. I'd still prefer that we export an enum / constants file that defined the options so they would fail when the module is loaded if one is removed and call sites are not updated.

Reply:
These strings are defined in https://github.com/gcanti/tcomb-form/blob/master/src/components.js#L64, so unfortunately, adding constants at the builder level won't solve that issue.